collator/src/collator/mod.rs

```diff
@@ -1547,21 +1549,41 @@ impl CollatorStdImpl {
             }
         };
 
+        let (last_imported_anchor_id, last_imported_chain_time) = self
+            .anchors_cache
+            .get_last_imported_anchor_id_and_ct()
+            .unwrap();
+        let prev_block_chain_time = working_state.prev_shard_data_ref().gen_chain_time();
+
+        // when no messages for processing
+        // then force empty shard block collation if related timeout elapsed
+        if matches!(
+            try_collate_check,
+            TryCollateCheck::ImportedAnchorHasNoExternals
+        ) {
+            let empty_sc_block_interval_ms =
```

Comment:
Perhaps using `Duration` would be  abetter approach here.

Reply:
We can't use Duration in blockchain configs due to different requirements of units for different params